CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Unreleased` header.
 
 # Unreleased
 
+- Fix crash when running iOS app on macOS.
```

Comment:
Nit: I think this is called running under "mac catalyst"?

Reply:
To be honest, I’m not even sure 😅 It sounds like Mac Catalyst is a way of building Mac apps from the same codebase as used for an iPad build. But that requires adding a Mac Catalyst target which I had not done. I just ran the iPad build directly on Mac. Maybe Apple also considers that part of the Catalyst